src/builder-flatpak-utils.c

```diff
@@ -1773,6 +1774,12 @@ parse_filesystem_flags (const char *filesystem, FlatpakFilesystemMode *mode)
       if (mode)
         *mode = FLATPAK_FILESYSTEM_MODE_CREATE;
     }
+  else if (g_str_has_suffix (filesystem, ":reset"))
+    {
+      len -= 6;
+      if (mode)
+        *mode = FLATPAK_FILESYSTEM_MODE_RESET;
+    }
```

Comment:
This makes it impossible to represent an app that has, for example, both `--nofilesystem=host:reset` and `--filesystem=host:ro`. Is that going to be a problem?

If it is, then I think we need to use the same trick as in Flatpak itself: convert `host:reset` into `{"host-reset": NONE}` in the hash table, and back to `--nofilesystem=host:reset` when serializing.

Reply:
I implemented what I suggest here in ed5e692.